keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ArtifactRepository.kt

```diff
@@ -75,6 +57,23 @@ interface ArtifactRepository : PeriodicallyCheckedRepository<DeliveryArtifact> {
     type: ArtifactType
   ): List<String>
 
+  /**
```

Comment:
I moved these down because they were in-between some `DeliveryArtifact`-related methods, but it makes the diff a bit harder to read. If you prefer, I can move them back where they were.